global_buffer/pwr/pts.tcl

```diff
@@ -0,0 +1,67 @@
+set search_path [list \
+  /tsmc16/TSMCHOME/digital/Front_End/timing_power_noise/NLDM/tphn16ffcllgv18e_110c/ \
+  /tsmc16/TSMCHOME/digital/Front_End/timing_power_noise/NLDM/tcbn16ffcllbwp16p90_100a/ \
+  /tsmc16/TSMCHOME/digital/Front_End/timing_power_noise/NLDM/tcbn16ffcllbwp16p90lvt_100a/ \
+  /tsmc16/TSMCHOME/digital/Front_End/timing_power_noise/NLDM/tcbn16ffcllbwp16p90ulvt_100a/ \
+  /tsmc16/TSMCHOME/digital/Front_End/timing_power_noise/NLDM/tcbn16ffcllbwp16p90pm_100a/ \
+  /sim/kongty/
+]                                                                                              
+set link_path [list \
+  * \
+  tphn16ffcllgv18ett0p8v1p8v85c.db \
+  tcbn16ffcllbwp16p90tt0p8v85c.db \
+  tcbn16ffcllbwp16p90lvttt0p8v85c.db \
+  tcbn16ffcllbwp16p90ulvttt0p8v85c.db \
+  tcbn16ffcllbwp16p90pmtt0p8v85c.db \
+  ts1n16ffcllsblvtc2048x64m8sw_130a_tt0p8v110c.db \
+]
+
+set power_enable_analysis true
+set power_analysis_mode time_based
+
+set sim_type "gate"
+set activity_file /sim/kongty/garnet/global_buffer/glb_pnr.vcd
+set parasitics_file {/sim/kongty/pnr_annotate/global_buffer.spef.gz /sim/kongty/pnr_annotate/glb_tile.spef.gz}
+read_verilog {/sim/kongty/pnr_annotate/global_buffer.lvs.v /sim/kongty/pnr_annotate/glb_tile.lvs.v}
+
+set dut "global_buffer"
+set report_dir "./reports"
+
+#########################################################
+# Read in design                                        #
+#########################################################
+current_design global_buffer
+set LINK_STATUS [link_design]
+ 
+#########################################################
+# Constraints                                           #
+#########################################################
+read_sdc ./constraints.tcl > "$report_dir/constraints.rpt"
+
+#########################################################
+# parasitics file
+#########################################################
+read_parasitics -format spef $parasitics_file
+
+#########################################################
+# Read VCD                                              #
+#########################################################
+#set time_window [list [1005000] [2285000]]
+switch [file extension $activity_file] {
+    ".vcd"  {read_vcd -rtl $activity_file -strip_path "top/dut" -time {417 544}}
+}
+
```

Comment:
In the long run maybe use ucli for the saif generation, dumping vcd slows down the simulation and vcd2saif takes a long time.

Reply:
Oh but you cannot do that for xrun. We need a runtime solution for saif generation for larger tests since vcds grow so quickly.

Reply:
That totally makes sense. As this PR is mostly about sim/synthesis, i think I will just keep it as it is now.
In the future PR regarding power, I can add runtime solution for saif generation instead of vcd.
Thanks for the comment.